src/apollo/client.js

```diff
@@ -12,7 +12,7 @@ export const client = new ApolloClient({
 
 export const txClient = new ApolloClient({
   link: new HttpLink({
-    uri: process.env.REACT_APP_TX_GRAPH_API_URL,
+    uri: process.env.REACT_APP_GRAPH_API_URL,
```

Comment:
Why did this change?

Reply:
The graph does not work and made this change to get transactions from furadao